apps/web/src/pages/quick-start/steps/Setup.tsx

```diff
@@ -21,13 +24,17 @@ import { When } from '../../../components/utils/When';
 import { colors } from '../../../design-system';
 import { getInAppActivated } from '../../../api/integration';
 import { useSegment } from '../../../components/providers/SegmentProvider';
+import { getApiKeys } from '../../../api/environment';
 
 export function Setup() {
   const [notificationTemplate, setNotificationTemplate] = useState<INotificationTemplate>();
   const { framework } = useParams();
   const { groups } = useNotificationGroup();
   const { templates = [], loading } = useTemplates();
   const { environment } = useEnvController();
+  const { data: apiKeys } = useQuery<{ key: string }[]>(['getApiKeys'], getApiKeys);
+  const apiKey = apiKeys?.length ? apiKeys[0].key : '';
```

Comment:
Are we always going to receive an array? If there are other potential types we could receive in this call might worth to check it is array rather than its length, as if receiving a string the condition would pass.

Reply:
At the moment it should be always an array, as it is on the environment entity.